src/replit/ai/modelfarm/google/language_models/text_embedding_model.py

```diff
@@ -1,54 +1,56 @@
 from dataclasses import dataclass
-from replit.ai.modelfarm.embedding_model import EmbeddingModel, EmbeddingModelResponse
-from typing import List, Dict, Any
+from typing import List
+
+from replit.ai.modelfarm import AsyncModelfarm, Modelfarm
+from replit.ai.modelfarm.structs.embeddings import Embedding, EmbeddingModelResponse
 
 
 @dataclass
 class TextEmbeddingStatistics:
-  token_count: float
-  truncated: bool
+    token_count: float
```

Comment:
A lot of the diff is formatting change, but I don't see a formatter configured in pyproject.toml. Could we add it as a dev dependency with a config (if necessary)?

Reply:
I develop on replit, so it was replit LSP